content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,140 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Recent tidymodels releases offer new functions for easier model building and
+    resampling, along with a new package for resampling spatial data.
+
+photo:
+  url: https://unsplash.com/photos/4Zk45jNyQS4
+  author: Timo Wielink
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
```

Comment:
Maybe we should add a new category for this: maybe roundup?

cc @batpigandme

Reply:
I like it

Reply:
If I just use it, will it be created?

Reply:
Yes, I think so.

Reply:
It did create it but it does not have an icon:

![Screen Shot 2021-03-11 at 3 44 57 PM](https://user-images.githubusercontent.com/12505835/110865381-c35cdd80-8280-11eb-8cad-d6d5d63a689f.png)

Reply:
Do we prefer "round-up" or "roundup"?

Reply:
These are both @batpigandme questions 😄

Reply:
Roundup (the noun) has been in use since 1790, so I'm pro-one-word. As for an icon, I think there are a few used for “summary” that kinda sorta fit (e.g. https://commons.wikimedia.org/wiki/File:Summary_icon.svg), but we could also just do like a little spiral, to stick with our minimalist look.

Reply:
Or, if we were feeling like cowboys, we could use a lasso, but I think that's got a strong connection to the selection tool for Photoshop/Illustrator users.

Reply:
Yes, I don't think a lasso 🤠 but I like either something like that little summary icon or a spiral.

Reply:
How [do the SVGs like `tidyCategory_.svg` get made](https://github.com/tidyverse/tidyverse.org/tree/master/themes/hugo-graphite/static/css/images)?